File: app/src/main/java/com/mobile/gitrepoapp/api/response/RepoCollectionModel.java
package com.mobile.gitrepoapp.api.response;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class RepoCollectionModel {

    @JsonProperty("total_count")
    public Integer totalCount;
    @JsonProperty("incomplete_results")
    public Boolean incompleteResults;
    @JsonProperty("items")
    public List<RepoDetailModel> items = null;

}

File: app/src/main/java/com/mobile/gitrepoapp/api/response/RepoCollectionModel.kt
package com.mobile.gitrepoapp.api.response

import com.fasterxml.jackson.annotation.JsonProperty

class RepoCollectionModel {
    @JsonProperty("total_count")
    var totalCount: Int? = null

    @JsonProperty("incomplete_results")
    var incompleteResults: Boolean? = null

    @JsonProperty("items")
    var items: List<RepoDetailModel> = arrayListOf()
}